doc/whats_new/v0.24.rst

```diff
@@ -580,6 +580,16 @@ Changelog
   with different endianness.
   :pr:`17644` by :user:`Qi Zhang <qzhang90>`.
 
+:mod:`sklearn.utils`
+....................
+
+- New :func:`utils._init_arpack_v0` which goal is to be used each time eigs,
```

Comment:
We normally do not include private functions in the whats new because it is user facing.

Instead, we can state what estimators have changed because of this updated, i.e. `PLSSVD`, etc.